src/actions/secrets.js

```diff
@@ -69,10 +69,12 @@ export function deleteSecret(secrets) {
       const deleteWithinTimePromise = Promise.race([response, timeout]);
       return deleteWithinTimePromise;
     });
-
     Promise.all(deletePromises)
       .then(() => {
-        dispatch({ type: 'SECRET_DELETE_SUCCESS' });
+        dispatch({
+          type: 'SECRET_DELETE_SUCCESS',
+          success: 'Secret(s) deleted successfully'
```

Comment:
Since the `subtitle` prop in `InlineNotification` is hardcoded, you really don't need to pass a string. With the type and reducer declared takes care of it.

Reply:
Ok, I've removed the ```success``` property for both create and delete